plugin/src/main/java/org/opensearch/ml/utils/TenantAwareHelper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.utils;
+
+import java.util.Objects;
+
+import org.opensearch.OpenSearchStatusException;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.ml.settings.MLFeatureEnabledSetting;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class TenantAwareHelper implements TenantAware {
```

Comment:
Usually "Helper" or "Utility" classes are stateless and full of static functions.

Why not add `tenantId` to method `validateTenantId` and `validateTenantResource` to make this class a real stateless helper class ?

Reply:
Sure.